ogc-csw/src/main/java/io/arlas/server/ogc/csw/CSWService.java

```diff
@@ -69,7 +70,9 @@
 import static io.arlas.server.utils.CheckParams.isBboxLatLonInCorrectRanges;
 
 
-public class CSWService extends CollectionRESTServices {
+public class CSWService {
+
+    public Logger LOGGER = LoggerFactory.getLogger(CSWService.class);
```

Comment:
LOGGER is not used

Reply:
Sorry, it's actually used

Reply:
it's a forgotten, we must remove these uses, it was for debugging